doc/source/tutorial/signal.rst

```diff
@@ -1756,26 +1760,148 @@ zeros, enlarging :math:`\vb{U}` so all slices which touch :math:`x[k]` contain
 the identical dual window
 
 .. math::
+   :label: eq_STFT_CanonDualWin
 
-    w_d[m] = w[m] \inv{\sum_{\eta\in\IZ} \big|w[m + \eta\, h]\big|^2}\ .
+    w_d[m] = w[m] \inv{\sum_{\eta\in\IZ} \big|w[m + \eta\, h]\big|^2}
+           =\ w[m] \inv{\sum_{l=0}^{M-1} \big|w[l]\big|^2 \delta_{l+m,\eta h}}\ ,
+             \quad \eta\in \IZ\ .
```

Comment:
Split over two (aligned-on-`=`) lines? It overflows for me at least

![image](https://github.com/user-attachments/assets/c451322d-4c63-42a2-98bf-cf6721bc3320)

Reply:
Done. I wish there would be scroll bars for formulas like for the code blocks. The problem is that in the current configuration the text width is allowed to vary to some amount (can be seen by changing the width of the browser window).